docs/roadmap.md

```diff
@@ -9,39 +9,16 @@ This document highlights some ideas for major features we'd like to implement in
 To get a more complete overview of planned features and current work, see the [issue tracker](https://github.com/cortexproject/cortex/issues).
 Note that these are not ordered by priority.
 
-## Helm charts and other packaging
+## CNCF Graduation Status
 
-We have a [helm chart](https://github.com/cortexproject/cortex-helm-chart) but it needs work before it can be effectively utilised by different backends. We also don't provide an official set of dashboards and alerts to our users yet. This is one of the most requested features and something we will tackle in the immediate future. We also plan on publishing debs, rpms along with guides on how to run Cortex on bare-metal.
+Cortex was accepted to the CNCF on September 20, 2018 and moved to the Incubating maturity level on August 20, 2020. The Cortex maintainers are working towards promoting the project to the graduation status. See [issue #6075](https://github.com/cortexproject/cortex/issues/6075) for tracking this progress.
 
-## Auth Gateway
+## Support for Prometheus Remote Write 2.0
 
-Cortex server has a simple authentication mechanism (X-Scope-OrgId) but users can't use the multitenancy features out of the box without complicated proxy configuration. It's hard to support all the different authentication mechanisms used by different companies but plan to have a simple but opinionated auth-gateway that provides value out of the box. The configuration could be as simple as:
+[Prometheus Remote Write 2.0](https://prometheus.io/docs/specs/remote_write_spec_2_0/)
 
-```
-tenants:
-- name: infra-team
-  password: basic-auth-password
-- name: api-team
-  password: basic-auth-password2
-```
+* adds a new Protobuf Message with new features enabling more use cases and wider adoption on top of performance and cost savings
+* deprecates the previous Protobuf Message from a 1.0 Remote-Write specification
+* adds mandatory X-Prometheus-Remote-Write-*-Written HTTP response headers for reliability purposes
 
-## Billing and Usage analytics
-
-We have all the metrics to track how many series, samples and queries each tenant is sending but don't have dashboards that help with this. We plan to have dashboards and UIs that will help operators monitor and control each tenants usage out of the box.
```

Comment:
I created https://github.com/cortexproject/cortex-jsonnet/issues/64 because this is nice. But it doesn't need to be part of the roadmap